connect/mirror/README.md

```diff
@@ -141,7 +141,40 @@ nearby clusters.
 N.B. that the `--clusters` parameter is not technically required here. MM2 will work fine without it; however, throughput may suffer from "producer lag" between
 data centers, and you may incur unnecessary data transfer costs.
 
-## Shared configuration
+## Configuration
+The following sections target for dedicated MM2 cluster. If running MM2 in a Connect cluster, please refer to KIP-382: MirrorMaker 2.0](https://cwiki.apache.org/confluence/display/KAFKA/KIP-382) for guidance.
```

Comment:
The link does not render, it's missing the opening `[`. 
Also the correct link to the KIP is https://cwiki.apache.org/confluence/display/KAFKA/KIP-382%3A+MirrorMaker+2.0

Reply:
resolved